src/base/static/components/atoms/typography.js

```diff
@@ -248,10 +248,10 @@ const ExternalLink = styled("a")({
   },
 });
 
-const InternalLink = styled(props => {
+const InternalLink = styled(({ children, className, href, ...rest }) => {
   return (
-    <Link to={props.href} className={props.className} {...props}>
-      {props.children}
+    <Link to={href} className={`mapseed-internal-link ${className}`} {...rest}>
```

Comment:
@Brian393 -- here's an example of how we can add stable classnames to components alongside the automatically generated classnames that change in production.